examples/plot_decoding_tutorial.py

```diff
@@ -65,8 +82,12 @@
 from nilearn.input_data import NiftiMasker
 masker = NiftiMasker(mask_img=mask_filename, standardize=True)
 
+###########################################################################
 # We give the masker a filename and retrieve a 2D array ready
 # for machine learning with scikit-learn
+# .. note::
+#	We can have a look at a report to see the mask computed by the 
+#	NiftiMasker by using :func:`masker.generate_report()``
```

Comment:
I think that you should actually run this command in a cell just above.

Reply:
I agree, actually I will change the explanation also. Seems not super clear.